### PR TITLE
fix(editor): set min value of 0 for min and max length in properties

### PR DESCRIPTION
**Pull request by philippfromme**
Related to camunda/camunda-modeler#2199

### CODE REVIEW

File: packages/form-js-editor/src/rendering/PropertiesPanel.js [hunk 1]
```diff
@@ -411,13 +426,15 @@ function ValidationGroup(field, editField) {
...
         label: 'Minimum Length',
+        min: 0,
         path: [ 'validate', 'minLength' ]
       }),
       NumberEntry({
         editField,
         field,
         id: 'maxLength',
         label: 'Maximum Length',
+        min: 0,
```
Comment (pinussilvestrus) [inline packages/form-js-editor/src/rendering/PropertiesPanel.js:437]: Does a max length of 0 sense? Shouldn't it be at least 1?

**Review by pinussilvestrus — COMMENTED** [review]
I can still set negative values via input (although not via the increase/decrease button anymore).

Kapture 2021-04-08 at 16 29 50

File: packages/form-js-editor/src/rendering/PropertiesPanel.js (hunk 1 above)
Comment (philippfromme) [reply packages/form-js-editor/src/rendering/PropertiesPanel.js:443]: Well, does a maximum length of 1 make sense? 😄 We shouldn't make any assumptions. Negative lengths are a bug but a length of 0 is not.

**Comment by philippfromme** [comment]
@pinussilvestrus Thanks for catching that. There's no ideal solution at the moment. I updated the behavior so it won't commit invalid just like the bpmn-js properties panel works. Unfortunately, there are is no error indication yet but at least you can't break your form fields anymore.

File: packages/form-js-editor/src/rendering/PropertiesPanel.js (hunk 1 above)
Comment (pinussilvestrus) [reply packages/form-js-editor/src/rendering/PropertiesPanel.js:443]: Makes sense. 👍

**Review by pinussilvestrus — APPROVED** [review]

**Pull request merged**